src/Agent.Worker/Release/ContainerFetchEngine/FetchEngine.cs

```diff
@@ -240,6 +240,11 @@ private void LogStatistics()
         {
             try
```

Comment:
can you log the file container URI of container to download before download begins also?